.eslintrc

```diff
@@ -21,5 +21,6 @@
     "no-use-before-define": ["warn"],
     "react/no-multi-comp": ["warn"],
     "react/jsx-no-bind": ["warn"],
+    "react/forbid-prop-types": ["off"]
```

Comment:
I don't like turning eslint's right off without good reasoning, bare minimum set this to `"warn"`